packages/app/src/app/overmind/internalActions.ts

```diff
@@ -257,8 +258,6 @@ export const setCurrentSandbox: AsyncAction<Sandbox> = async (
 
   effects.executor.setupExecutor();
 
-  effects.fsSync.syncCurrentSandbox();
```

Comment:
Should this be removed? I think whenever the sandbox changes we need to send the new files to the worker. Oh wait, we look at the structure change now, so it happens automatically... Okay, nice! Keeping this comment here for documentation purposes.

Reply:
Jep, exactly :)